calculater/app/src/main/java/com/example/calculater/CalculaterModel.kt

```diff
@@ -37,4 +37,18 @@ class CalculaterModel{
 
     }
 
+    fun clear() {
+        numbers.clear()
+        operators.clear()
+        lastNumber = ""
+    }
+
+    fun checkZeroDiv(value: Double): Boolean {
+        return if(operators.size > 0){
```

Comment:
return if (operators.isNotEmpty) {
(operators.last() == Operators.DIV && value.toInt() == 0).not()
} else {
  true
}